src/timepicker/timepicker.ts

```diff
@@ -136,7 +159,7 @@ export class NgbTimepicker implements ControlValueAccessor,
    */
   @Input() size: 'small' | 'medium' | 'large';
 
-  constructor(config: NgbTimepickerConfig, private _ngbTimeAdapter: NgbTimeAdapter<any>) {
+  constructor(private readonly config: NgbTimepickerConfig, private _ngbTimeAdapter: NgbTimeAdapter<any>) {
```

Comment:
As config becomes private, please rename to `_config` (see https://github.com/ng-bootstrap/ng-bootstrap/wiki/Contributions%3A-Code-conventions)